FreeRTOS/FreeRTOS-Plus-TCP
# [IPv6] Fix IPv4 TCP.

[IPv6] Fix IPv4 TCP.

Description
-----------

- Fix IPv4 TCP.
- Replace ipLOCAL_IP_ADDRESS_POINTER with endpoint IP address.

Test Steps
-----------

Run IPv4 datagram tests.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
